pkgs/development/libraries/libmbim/default.nix

```diff
@@ -29,9 +30,12 @@ stdenv.mkDerivation rec {
 
   mesonFlags = [
     "-Dudevdir=${placeholder "out"}/lib/udev"
-    (lib.mesonBool "introspection" withIntrospection)
+    (lib.mesonBool "introspection" true)
+    (lib.mesonBool "man" withDocs)
```

Comment:
Perhaps make `help2man` dependency conditional as well

Reply:
I did, thanks for mentioning.